pkg/clusterversion/cockroach_versions.go

```diff
@@ -197,15 +197,17 @@ const (
 	// was introduced after this version was first introduced. Later code in the
 	// release relies on the job to run the migration but the job relies on
 	// its startup migrations having been run. Versions associated with long
-	// running migrations must follow LongRunningMigrations.
+	// running migrations must follow deletedLongRunningMigrations.
 	replacedTruncatedAndRangeAppliedStateMigration
 	// replacedPostTruncatedAndRangeAppliedStateMigration is like the above
 	// version. See its comment.
 	replacedPostTruncatedAndRangeAppliedStateMigration
-	// LongRunningMigrations introduces the LongRunningMigrations table and jobs.
+	// deletedLongRunningMigrations stands in for LongRunningMigrations which
+	// was deleted after the 21.1 release. LongRunningMigrations introduced
+	// the LongRunningMigrations table and jobs.
 	// All versions which have a registered long-running migration must have a
 	// version higher than this version.
-	LongRunningMigrations
+	deletedLongRunningMigrations
```

Comment:
Why not just delete these?

Reply:
Can't remember why, now I don't see why not, so: will do.